### DISCUSSION TITLE
which key to expand or collapse header ?

### DISCUSSION THREAD

**Discussion by lost22git** [nvim-neorg/neorg#679] [General]
which key to expand or collapse header ?

**Comment by max397574** [comment]
just use the normal fold keybindings (zc, zo)
this uses treesitter so it's quite unstable atm
a `:e` fixes it most times

**Reply by Apeiros-46B** [reply]
you can also use `zx` to force update folds

**Reply by max397574** [reply]
doesn't work for me
this could be because the ast needs to be regenerated or sth (idk how this internally works in treesitter) and not just the foldexpr needs to be reevaluated
